### PR TITLE
Onboard the IaaS network `delete` command and extend `project create`

### PR DESCRIPTION
**Pull request by DiogoFerrao**

### CODE REVIEW

File: internal/cmd/beta/network-area/delete/delete.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+				return fmt.Errorf("delete network area: %w", err)
+			}
+
+			// Wait for async operation, if async mode not enabled
+			if !model.Async {
+				s := spinner.New(p)
+				s.Start("Deleting network area")
+				_, err = wait.DeleteNetworkAreaWaitHandler(ctx, apiClient, *model.OrganizationId, model.AreaId).WaitWithContext(ctx)
+				if err != nil {
+					return fmt.Errorf("wait for Redis instance deletion: %w", err)
```
Comment (joaopalet) [inline internal/cmd/beta/network-area/delete/delete.go:85]:
```suggestion
					return fmt.Errorf("wait for network area deletion: %w", err)
```

File: internal/cmd/beta/network-area/delete/delete.go
Comment (joaopalet) [inline internal/cmd/beta/network-area/delete/delete.go:1]: DO you know what happens when we try to delete a network area that is used in projects? Does it error?

File: internal/cmd/project/create/create.go [hunk 2]
```diff
@@ -94,6 +100,7 @@ func configureFlags(cmd *cobra.Command) {
 	cmd.Flags().String(parentIdFlag, "", "Parent resource identifier. Both container ID (user-friendly) and UUID are supported")
 	cmd.Flags().String(nameFlag, "", "Project name")
 	cmd.Flags().StringToString(labelFlag, nil, "Labels are key-value string pairs which can be attached to a project. A label can be provided with the format key=value and the flag can be used multiple times to provide a list of labels")
+	cmd.Flags().Var(flags.UUIDFlag(), networkAreaIdFlag, "Network area ID")
```
[the remark this answers is not in the document]
Comment (DiogoFerrao) [reply internal/cmd/project/create/create.go]: Good point, thanks for the tip. I added the help to the command description and improved the flag description to include the naming of the STACKIT Network Area

File: internal/cmd/beta/network-area/delete/delete.go
Comment (DiogoFerrao) [reply internal/cmd/beta/network-area/delete/delete.go:1]: I think documenting it is the best approach right? There is nothing we can do about it

Comment (joaopalet) [reply internal/cmd/beta/network-area/delete/delete.go:1]: Yes, that's perfectly fine 👍

**Review by joaopalet — APPROVED** [review]

**Pull request merged**